File: parcels/foo/foo.py
def main(use: callable, **kwargs) -> dict:
    """."""
    Base = use("/tools/base.py")

    class Foo(Base):
        def __init__(self):
            Base.__init__(self, foo="Py foo")
           

        @property
        def foo(self):
            return self._["foo"]
        
    def foo():
        print('foo')

    
    return Foo, foo
    return dict(Foo=Foo, foo=foo)

File: parcels/foo/foo.js
export default async (use) => {
  const { component } = await use("@/rollo/");
  const Base = use("@@/base/base.js");
  const foo = use("@@/foo/foo.py").foo;

  foo();

  class Foo extends Base {
    #_ = {};
    constructor() {
      super();
      this._.foo = "JS foo";
    }

    get foo() {
      return this._.foo;
    }
  }

  return { Foo };
};
